Fix broken unit test

`grpc_handler_test.go` has syntax error.

```
# 0chain.net/blobbercore/handler [0chain.net/blobbercore/handler.test]
./grpc_handler_test.go:39:35: not enough arguments in call to NewServerWithMiddlewares
	have ()
	want ("github.com/grpc-ecosystem/go-grpc-middleware/ratelimit".Limiter)
FAIL	0chain.net/blobbercore/handler [build failed]
```

--- a/code/go/0chain.net/blobbercore/go.mod
+++ b/code/go/0chain.net/blobbercore/go.mod
@@ -19,6 +19,7 @@ require (
 	github.com/remeh/sizedwaitgroup v0.0.0-20180822144253-5e7302b12cce
 	github.com/spf13/viper v1.7.0
 	github.com/stretchr/testify v1.7.0
+	go.uber.org/ratelimit v0.2.0
 	go.uber.org/zap v1.16.0
 	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
 	google.golang.org/genproto v0.0.0-20210224155714-063164c882e6

--- a/code/go/0chain.net/blobbercore/handler/grpc_handler_test.go
+++ b/code/go/0chain.net/blobbercore/handler/grpc_handler_test.go
@@ -11,6 +11,8 @@ import (
 	"testing"
 	"time"
 
+	rl "go.uber.org/ratelimit"
+
 	"0chain.net/blobbercore/allocation"
 	"0chain.net/blobbercore/blobbergrpc"
 	"0chain.net/blobbercore/datastore"
@@ -36,7 +38,7 @@ var (
 
 func startGRPCServer(t *testing.T) {
 	lis = bufconn.Listen(1024 * 1024)
-	grpcS := NewServerWithMiddlewares()
+	grpcS := NewServerWithMiddlewares(&common.GRPCRateLimiter{Limiter: rl.New(int(1000))})
 	RegisterGRPCServices(mux.NewRouter(), grpcS)
 	go func() {
 		if err := grpcS.Serve(lis); err != nil {

